app/lib/csv/buyers_exporter.rb

```diff
@@ -24,10 +25,10 @@ def values_for_account(account)
       account.bought_account_plan.try!(:name),
       account.created_at.to_s(:db),
       account.bought_cinstances.count,
-      account.first_admin.try!(:display_name),
-      account.first_admin.try!(:email),
+      account.decorate.admin_user_display_name,
+      user&.email,
       account.extra_fields.to_json,
-      account.first_admin.try!(:extra_fields).to_json
+      user&.extra_fields.to_json
```

Comment:
user should not be `nil`, you probably meant

```suggestion
      user.extra_fields.to_json
```

Reply:
let's move this conversation to https://github.com/3scale/porta/pull/2024/files#r453490457